src/client/index.css

```diff
@@ -1,4 +1,4 @@
-@import url('https://fonts.googleapis.com/css?family=Montserrat&display=swap');
+/* @import url('https://fonts.googleapis.com/css?family=Montserrat&display=swap');
```

Comment:
Since you commented this part your app is not media responsive. You should use @media rule in you main section. On Main Page I can se that text disapears and image is out of vieport range on smaller screens.